Makefile

```
@@ -282,6 +282,18 @@ else
 	npx react-native run-ios
 endif
 
+show-ios-devices: export TARGET := ios
+show-ios-devices: ##@other shows connected ios device and its name
+	xcrun xctrace list devices
+
+run-ios-device: export TARGET := ios
+run-ios-device: ##@run iOS app and start it on a connected device by its name
+ifneq ("$(DEVICE_NAME)", "")
+	react-native run-ios --device "$(DEVICE_NAME)"
+else
+	echo "Please provide device name with DEVICE_NAME flag like this: make run-ios-device DEVICE_NAME=your-device-name"
```

You can use an `make` error here:
```suggestion
	$(error Usage: make run-ios-device DEVICE_NAME=your-device-name"
```

---

oh let me try that out :)